templates/provider/kcm-templates/Chart.yaml

```diff
@@ -13,4 +13,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.0.1
```

Comment:
This change should not be presented. Only the corresponding cluster template version is to be bumped.